helm/charts/kratos/values.yaml

```diff
@@ -185,6 +185,13 @@ deployment:
   # - --sqa-opt-out
   extraArgs: []
 
+  # Set these to override the entrypoint of the deployment container
+  # (e.g. to source dynamic secrets or export environment dynamic variables)
+  overrideArgs:
+    command: []
```

Comment:
Not sure if an array is a good idea here. For more complex scenarios, passing a multiline string like 
```yaml
command: |
  - some very complex
  - stuff that requires {{ .Values.other.stuff }}
```

and then calling `toYaml | tpl` might be better

Reply:
Please refer to the [Kartos chart](https://github.com/ory/k8s/blob/master/helm/charts/kratos/templates/deployment.yaml#L47-L51) how we handle similar parsing issues and how to [use it in values](https://github.com/ory/k8s/blob/master/.circleci/values/kratos.yaml#L83-L92)